resources/assets/js/videos/components/videoScreen/videoPlayback.vue

```diff
@@ -153,6 +153,33 @@ export default {
         handleSeeked() {
             this.renderVideo(true);
         },
+        // 3 next methods are a workaround to get previous and next frames, adapted from here: https://github.com/angrycoding/requestVideoFrameCallback-prev-next/tree/main
+        frameInfoCallback() {
+            let promise = new Vue.Promise((resolve, reject) => {
+                if ("requestVideoFrameCallback" in HTMLVideoElement.prototype) {
+                    this.video.requestVideoFrameCallback((now, metadata) => {
+                        resolve(metadata);
+                    })
+                }
+                else {
+                    reject("Your browser does not support requestVideoFrameCallback().");
+                }
+            })
+            return promise;
+        },
+        // Methods to jump back and forward in video. Step is given by parameter jumpStep.
+        async jumpBackward() {
+            if (this.video.currentTime > 0 && this.jumpStep > 0) {
+                this.video.currentTime -= this.jumpStep;
+                await this.frameInfoCallback();
+            }
+        },
+        async jumpForward() {
+            if (!this.video.ended && this.jumpStep > 0) {
+                this.video.currentTime += this.jumpStep;
+                await this.frameInfoCallback();
+            }
+        },
```

Comment:
Why do you need the `frameInfoCallback` here? The video time should change by setting `currentTime` only.

Reply:
Yes you are right it's useless here, I removed it.